saleor/graphql/order/bulk_mutations/draft_orders.py

```diff
@@ -0,0 +1,20 @@
+import graphene
+
+from ....order import models
+from ...core.mutations import ModelBulkDeleteMutation
+
+
+class DraftOrderBulkDelete(ModelBulkDeleteMutation):
+    class Arguments:
+        ids = graphene.List(
+            graphene.ID,
+            required=True,
+            description='List of draft order IDs to delete.')
+
+    class Meta:
+        description = 'Deletes draft orders.'
+        model = models.Order
```

Comment:
Since this takes just the list of IDs it would only allow deleting regular orders, is that right? If so, we should add some checks to prevent that. Only draft orders can be deleted in Saleor, while regular orders can be at most canceled.